store/postgres/migrations/2023-08-23-143628_add_handlers_to_subgraph_features/up.sql

```diff
@@ -0,0 +1,6 @@
+truncate table subgraphs.subgraph_features;
```

Comment:
truncating the table so that the new columns values will get populated, `subgraph_features` table is populated during `build_subgraph_runner` so it will repopulate all the entries on startup